app/src/main/java/org/mozilla/focus/Inject.java

```diff
@@ -34,14 +53,10 @@ public static void enableStrictMode() {
         final StrictMode.ThreadPolicy.Builder threadPolicyBuilder = new StrictMode.ThreadPolicy.Builder().detectAll();
         final StrictMode.VmPolicy.Builder vmPolicyBuilder = new StrictMode.VmPolicy.Builder().detectAll();
 
-        if (AppConstants.isBetaBuild()) {
```

Comment:
We don't check the build type and always popup dialog with log?

Reply:
if it's Release build, we'll return immediately.
if it's Beta / debug / firebase build, we'll always popup dialogue with log
in AndroidTest, we only log error